src/components/Results/Statistics.vue

```diff
@@ -1,62 +1,53 @@
 <template>
   <div :class="$style.container">
-    <div v-for="(data, i) in countedData" :key="i">
-      <view-tab :counted-data="data" @copy="copy(data)" />
+    <div v-for="(data, i) in resultsPerQuestion.questions" :key="i">
```

Comment:
```suggestion
    <div v-for="question in resultsPerQuestion.questions" :key="question.id">
```
とかのほうが変数名がすっきりすると思います